main/integration-tests/__test__/api-tests/users/bulk-add-users.test.js

```diff
@@ -57,10 +57,19 @@ describe('Create user scenarios', () => {
       });
     });
 
-    it('should fail for adding user that already exist', async () => {
+    it('should fail with badRequest code for adding users that already exist', async () => {
       const admin1Session = await setup.createAdminSession();
       const newUser = admin1Session.resources.users.defaults();
       await expect(admin1Session.resources.users.bulkAddUsers([defaultUser, newUser])).rejects.toMatchObject({
+        code: errorCode.http.code.badRequest,
```

Comment:
Can we add new unit tests for badRequest and internalError scenario as well?

Reply:
just added!